src/mountscript.sh

```diff
@@ -215,15 +224,56 @@ get_dir_from_share()
     IFS=: read _ share <<< "$hostshare"
     IFS=/ read _ account container extra <<< "$share"
 
-    if [ -z "$account" -o -z "$container" -o -n "$extra" ]; then
```

Comment:
Just realize v3 helper don't want to do sub-mount, you need to have different logic for v3 and v4

Reply:
handled it